src/components/SchemaTable/index.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: 0;
+  margin: 0;
+  padding: 0;
+}
+p {
+  white-space: nowrap;
+}
+.wrapper {
+  display: grid;
+  grid-template-columns: [left-panel] 1fr [right-panel] 1fr;
+  row-gap: 4px;
+}
+.left-panel {
+  background-color: lightyellow;
+  overflow-x: auto;
+}
+.right-panel {
+    overflow-x:auto;
+}
```

Comment:
material-ui comes with CSS-in-JS built-in and helps users override these styles more easily. Let's use that rather than rely on CSS. https://material-ui.com/styles/basics/

Reply:
Applied changes to use material-ui's Hook API for CSS-in-JS styling! 🚀 

I've noticed that in Taskcluster, the Higher-order Component API is used, however, recently material-ui has added a new hook API method for styling. Since they all share the same underlying logic, I used the hook API for a cleaner style of code and consistency with the overall hook api in react :)